lib/sqlalchemy/ext/asyncio/engine.py

```diff
@@ -82,6 +82,21 @@ def create_async_engine(url: Union[str, URL], **kw: Any) -> AsyncEngine:
             "streaming result set"
         )
     kw["_is_async"] = True
+    async_creator = kw.pop("async_creator", None)
+    if async_creator:
+
+        async def wrap_async_creator() -> Any:
```

Comment:
I'm not sure there is a need for this. What advantage does it have compared to just passing `async_creator` to `creator_fn`?

Reply:
good call, removed unnecessary `wrap_async_creator` function and passed `async_creator` directly.